substrate/primitives/core/src/ecdsa.rs

```diff
@@ -328,7 +325,7 @@ impl Signature {
 	/// Recover the public key from this signature and a message.
 	#[cfg(feature = "full_crypto")]
 	pub fn recover<M: AsRef<[u8]>>(&self, message: M) -> Option<Public> {
-		self.recover_prehashed(&blake2_256(message.as_ref()))
+		self.recover_prehashed(&sp_crypto_hashing::blake2_256(message.as_ref()))
```

Comment:
nit: `sp_crypto_hashing::blake2_256` is used multiple times, maybe better keep import?
```suggestion
		self.recover_prehashed(&blake2_256(message.as_ref()))
```